crates/config/README.md

```diff
@@ -0,0 +1,97 @@
+# Config
+
+The library that contains configuration information for a Telcoin Network node.
+
+## Purpose & Scope
+
+The config crate contains various configuration parameters that allow a node to participate with the protocol.
+It contains sensitive information and manages private keys.
+
+## Key Components
+
+### ConsensusConfig
+
+The configuration for consensus layer.
+The `Committee` and `WorkerCache` contains all information for committee participates that vote to extend the canonical tip.
+The committee is comprised of staked validators and information to verify their communication.
+
+### GenesisConfig
+
+The configuration for network genesis.
+This is only used once, at the network's genesis.
+The initial committee is generated by a "Master of Ceremony" using a centralized VCS (ie - GitHub).
+Each validator must submit a proof-of-possession, which is validated using this logic.
+
+See `bin/telcoin-network/genesis` for more information on how this is used.
+
+### KeyConfig
+
+The most critical config for a node's security.
+This contains the node's keys used to authenticate protocol messages.
+
+The protocol requires access to three keys in the current version:
+
+- Primary BLS12-381 key
+- Primary Ed25519 network key
+- Worker Ed25519 network key (only support 1 worker per node for now)
```

Comment:
It is probably worth mentioning in this section that the BLS private key is not exposed outside KeyConfig while the network private keys are so that they can be used with libp2p.  The BLS key is the more important key to impersonate a validator and network keys can theoretically be rotated (although we don't do that yet).